src/config/window_definition.rs

```diff
@@ -94,20 +105,29 @@ impl std::str::FromStr for Side {
             "r" | "right" => Ok(Side::Right),
             "t" | "top" => Ok(Side::Top),
             "b" | "bottom" => Ok(Side::Bottom),
+            "c" | "center" => Ok(Side::Center),
+            "tl" | "top-left" => Ok(Side::TopLeft),
+            "tr" | "top-right" => Ok(Side::TopRight),
+            "bl" | "bottom-left" => Ok(Side::BottomLeft),
+            "br" | "bottom-right" => Ok(Side::BottomRight),
             _ => Err(anyhow!(
-                "Failed to parse {} as valid side. Must be one of \"left\", \"right\", \"top\", \"bottom\"",
+                "Failed to parse {} as valid side. Must be one of \"left\", \"right\", \"top\", \"bottom\", \"top-right\", \
+                 \"top-left\", \"bottom-left\", \"bottom-right\"",
```

Comment:
You can use a raw string thingy here to get rid of all of the escaping - do `r#"Failed to parse ...... "left", "right", ......."#`, that get's rid of all of the `\"`